src/javascript/app_2/Stores/base_store.js

```diff
@@ -139,9 +146,9 @@ export default class BaseStore {
         });
 
         if (storage === BaseStore.STORAGES.LOCAL_STORAGE) {
-            localStorage.setItem(this.constructor.name, snapshot);
+            localStorage.setItem(this.store_name ? this.store_name : this.constructor.name, snapshot);
```

Comment:
@OskarAhl  if you check the validity of `store_name` in the constructor it doesn't need to check it here anymore.